cmd/ctr/commands/tasks/metrics.go

```diff
@@ -24,15 +24,16 @@ import (
 	"text/tabwriter"
 
 	wstats "github.com/Microsoft/hcsshim/cmd/containerd-shim-runhcs-v1/stats"
-	v1 "github.com/containerd/cgroups/stats/v1"
-	v2 "github.com/containerd/cgroups/v2/stats"
+	v1 "github.com/containerd/cgroups/v3/cgroup1/stats"
+	v2 "github.com/containerd/cgroups/v3/cgroup2/stats"
```

Comment:
I personally like this symmetric naming :)

Reply:
SGTM